File: module-user/src/main/java/com/zhf/user/UserModuleInit.java
package com.zhf.user;

import com.zhf.base.base.BaseApplication;
import com.zhf.common.IModuleInit;

/**
 * 应用模块:
 * <p>
 * 类描述:
 * <p>
 *
 */
public class UserModuleInit implements IModuleInit {
    @Override
    public boolean onInitAhead(BaseApplication application) {
        return false;
    }

    @Override
    public boolean onInitLow(BaseApplication application) {
        return false;
    }
}

File: module-user/src/main/java/com/zhf/user/UserModuleInit.kt
package com.zhf.user

import com.zhf.base.base.BaseApplication
import com.zhf.common.IModuleInit

/**
 * created by demoless on 2021/1/29
 * description:
 */
class UserModuleInit : IModuleInit {
    override fun onInitAhead(application: BaseApplication?): Boolean = false

    override fun onInitLow(application: BaseApplication?): Boolean = false
}